packages/ui/src/atom/circular-loader-atom/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './circular-loader-atom.vue';
```

Comment:
`index.js` not used for atoms in ui-kit, please drop it

Reply:
fixed